src/conf_mode/service_ids_fastnetmon.py

```diff
@@ -55,8 +55,11 @@ def verify(fastnetmon):
     if 'mode' not in fastnetmon:
         raise ConfigError('Specify operating mode!')
 
-    if 'listen_interface' not in fastnetmon:
-        raise ConfigError('Specify interface(s) for traffic capture')
+    if fastnetmon.get('mode') == 'mirror' and 'listen_interface' not in fastnetmon:
+        raise ConfigError('Specify interface(s) for traffic capture mode mirror')
+
+    if fastnetmon.get('mode') == 'sflow' and 'listen_address' not in fastnetmon.get('sflow', {}):
+        raise ConfigError('Specify sflow "listen-address" for mode sflow')
```

Comment:
I'd say "Incorrect settings for 'mode sflow': must specify sFlow listen address".

Reply:
Done!